src/main/java/org/xstefank/check/TemplateChecker.java

```diff
@@ -91,4 +93,12 @@ private static FormatConfig readConfig() {
         }
     }
 
+    private boolean shouldSkip(JsonNode payload) {
+        if (config.getFormat().getSkippingPatterns() != null) {
```

Comment:
you are checking both payload and skippingPatterns != null also in the call on line 99